### ISSUE TITLE
Bug: clones + yarn install fails on rollup stage

### ISSUE THREAD

**Issue by orta** [styled-components/styled-components#1121]
## Version
2.1.0 - af7975c

## Reproduction

```
~/d/p/oss $ git clone https://github.com/styled-components/styled-components.git
Cloning into 'styled-components'...
remote: Counting objects: 7961, done.
remote: Compressing objects: 100% (53/53), done.
remote: Total 7961 (delta 30), reused 47 (delta 17), pack-reused 7890
Receiving objects: 100% (7961/7961), 2.61 MiB | 160.00 KiB/s, done.
Resolving deltas: 100% (5257/5257), done.

 ~/d/p/oss $ cd styled-components/                                                                                                                                                                            

 ~/d/p/o/styled-components $ yarn
yarn install v0.27.5
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning "babel-loader@6.4.1" has unmet peer dependency "webpack@1 || 2 || ^2.1.0-beta || ^2.2.0-rc".
warning "react-native@0.39.2" has incorrect peer dependency "react@~15.4.0-rc.4".
[4/4] Building fresh packages...
success Saved lockfile.

$ npm run build

> styled-components@2.1.2 build /Users/orta/dev/projects/oss/styled-components
> npm run build:lib && npm run build:dist


> styled-components@2.1.2 prebuild:lib /Users/orta/dev/projects/oss/styled-components
> rimraf lib/*


> styled-components@2.1.2 build:lib /Users/orta/dev/projects/oss/styled-components
> babel --out-dir lib --ignore "*.test.js" src

src/constructors/constructWithOptions.js -> lib/constructors/constructWithOptions.js
src/constructors/css.js -> lib/constructors/css.js
src/constructors/injectGlobal.js -> lib/constructors/injectGlobal.js
src/constructors/keyframes.js -> lib/constructors/keyframes.js
src/constructors/styled.js -> lib/constructors/styled.js
src/hoc/withTheme.js -> lib/hoc/withTheme.js
src/index.js -> lib/index.js
src/models/BrowserStyleSheet.js -> lib/models/BrowserStyleSheet.js
src/models/ComponentStyle.js -> lib/models/ComponentStyle.js
src/models/InlineStyle.js -> lib/models/InlineStyle.js
src/models/ServerStyleSheet.js -> lib/models/ServerStyleSheet.js
src/models/StyleSheet.js -> lib/models/StyleSheet.js
src/models/StyleSheetManager.js -> lib/models/StyleSheetManager.js
src/models/StyledComponent.js -> lib/models/StyledComponent.js
src/models/StyledNativeComponent.js -> lib/models/StyledNativeComponent.js
src/models/ThemeProvider.js -> lib/models/ThemeProvider.js
src/native/index.js -> lib/native/index.js
src/no-parser/css.js -> lib/no-parser/css.js
src/no-parser/flatten.js -> lib/no-parser/flatten.js
src/no-parser/index.js -> lib/no-parser/index.js
src/no-parser/stringifyRules.js -> lib/no-parser/stringifyRules.js
src/primitives/index.js -> lib/primitives/index.js
src/test/utils.js -> lib/test/utils.js
src/types.js -> lib/types.js
src/utils/create-broadcast.js -> lib/utils/create-broadcast.js
src/utils/createWarnTooManyClasses.js -> lib/utils/createWarnTooManyClasses.js
src/utils/domElements.js -> lib/utils/domElements.js
src/utils/extractCompsFromCSS.js -> lib/utils/extractCompsFromCSS.js
src/utils/flatten.js -> lib/utils/flatten.js
src/utils/generateAlphabeticName.js -> lib/utils/generateAlphabeticName.js
src/utils/getComponentName.js -> lib/utils/getComponentName.js
src/utils/interleave.js -> lib/utils/interleave.js
src/utils/isStyledComponent.js -> lib/utils/isStyledComponent.js
src/utils/isTag.js -> lib/utils/isTag.js
src/utils/once.js -> lib/utils/once.js
src/utils/stringifyRules.js -> lib/utils/stringifyRules.js
src/utils/validAttr.js -> lib/utils/validAttr.js
src/vendor/glamor/hash.js -> lib/vendor/glamor/hash.js
src/vendor/postcss/at-rule.js -> lib/vendor/postcss/at-rule.js
src/vendor/postcss/comment.js -> lib/vendor/postcss/comment.js
src/vendor/postcss/container.js -> lib/vendor/postcss/container.js
src/vendor/postcss/css-syntax-error.js -> lib/vendor/postcss/css-syntax-error.js
src/vendor/postcss/declaration.js -> lib/vendor/postcss/declaration.js
src/vendor/postcss/input.js -> lib/vendor/postcss/input.js
src/vendor/postcss/lazy-result.js -> lib/vendor/postcss/lazy-result.js
src/vendor/postcss/list.js -> lib/vendor/postcss/list.js
src/vendor/postcss/node.js -> lib/vendor/postcss/node.js
src/vendor/postcss/parse.js -> lib/vendor/postcss/parse.js
src/vendor/postcss/parser.js -> lib/vendor/postcss/parser.js
src/vendor/postcss/postcss.js -> lib/vendor/postcss/postcss.js
src/vendor/postcss/processor.js -> lib/vendor/postcss/processor.js
src/vendor/postcss/result.js -> lib/vendor/postcss/result.js
src/vendor/postcss/root.js -> lib/vendor/postcss/root.js
src/vendor/postcss/rule.js -> lib/vendor/postcss/rule.js
src/vendor/postcss/stringifier.js -> lib/vendor/postcss/stringifier.js
src/vendor/postcss/stringify.js -> lib/vendor/postcss/stringify.js
src/vendor/postcss/terminal-highlight.js -> lib/vendor/postcss/terminal-highlight.js
src/vendor/postcss/tokenize.js -> lib/vendor/postcss/tokenize.js
src/vendor/postcss/vendor.js -> lib/vendor/postcss/vendor.js
src/vendor/postcss/warn-once.js -> lib/vendor/postcss/warn-once.js
src/vendor/postcss/warning.js -> lib/vendor/postcss/warning.js
src/vendor/postcss-nested/index.js -> lib/vendor/postcss-nested/index.js
src/vendor/postcss-safe-parser/parse.js -> lib/vendor/postcss-safe-parser/parse.js
src/vendor/postcss-safe-parser/safe-parser.js -> lib/vendor/postcss-safe-parser/safe-parser.js

> styled-components@2.1.2 prebuild:dist /Users/orta/dev/projects/oss/styled-components
> rimraf dist/*


> styled-components@2.1.2 build:dist /Users/orta/dev/projects/oss/styled-components
> rollup -c && rollup -c --environment ESBUNDLE && rollup -c --environment PRODUCTION

Creating development UMD bundle
rollup-plugin-inject: failed to parse /Users/orta/dev/projects/oss/styled-components/src/models/StyledComponent.js. Consider restricting the plugin to particular files via options.include
Creating ES modules bundle...
rollup-plugin-inject: failed to parse /Users/orta/dev/projects/oss/styled-components/src/models/StyledComponent.js. Consider restricting the plugin to particular files via options.include
Creating production UMD bundle...
rollup-plugin-inject: failed to parse /Users/orta/dev/projects/oss/styled-components/src/models/StyledComponent.js. Consider restricting the plugin to particular files via options.include
Done in 76.21s.

 ~/d/p/o/styled-components $ node -v 
v8.1.4
````

## Steps to reproduce

```sh
git clone https://github.com/styled-components/styled-components.git
cd styled-components
yarn
```
## Behavior

The `yarn` command does does  install correctly, however `build:dist` fails. Is this critical? Nah, but I know its the kind of thing that you don't notice if you don't have fresh eyes to a project 👍

**Comment by mxstbr** [comment]
Thanks! I thought I fixed this in #1119, when did you clone? 🤔

**Comment by orta** [comment]
Straight before posting this issue - af7975c - which is still master's head

**Comment by bkazi** [comment]
This might not necessarily be related
but I had this issue few days ago even with the fix from #1119 included
However since updating to yarn 1.0.1 I reinstalled node dependencies and it has disappeared
I notice you're on a pre 1.0.0 version of yarn so maybe that's the difference, bit of a long shot really

**Comment by mxstbr** [comment]
I cna't replicate this anymore, so I'm going to close it. 👌
